.github/workflows/pull-request.yaml

```diff
@@ -27,8 +27,10 @@ jobs:
           allowlist: |
             actions/checkout
             actions/setup-go
+            actions/upload-artifact
 
   lint:
+    needs: [verify-actions]
```

Comment:
Instead of creating a dependency between verify-actions and the other jobs, I thought about moving this into a separate workflow that only runs if a GitHub workflow changed, since this is the only trigger relevant for this.

Reply:
https://github.com/kyma-project/istio/pull/210/commits/8ee882fd6e61b7c9e955cc5b5fbb82e4a82dcdff